packages/nepal/README.md

```diff
@@ -0,0 +1,31 @@
+# 🇳🇵 Nepal
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-blue.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+Contains all the Provinces of Nepal:
+* Sudurpashchim Pradesh
+* Karnali Pradesh
+* Province No. 5
+* Gandaki Pradesh
+* Bagmati Pradesh
+* Province No. 2
+* Province No. 1
+
+## Installation
+
+### npm
+
+`npm install --save @svg-maps/nepal`
+
+### yarn
+
+`yarn add @svg-maps/nepal`
+
+## Credits
+
+This map is based on Government of Nepal, Ministry of Land Management, Cooperatives and Poverty Alleviation  (http://dos.gov.np/).
```

Comment:
Could you add a link to the original map with the license information please? Or did you create it yourself?